Tasks/MavenV2/Tests/L0RestoreOriginalPomXml.ts

```diff
@@ -0,0 +1,148 @@
+        'pom.xml': true
+    },
+    exec: {
+        '/home/bin/maven/bin/mvn -version': {
+            code: 0,
+            stdout: 'Maven version 1.0.0'
+        },
+        '/home/bin/maven/bin/mvn -f pom.xml help:effective-pom': {
+            code: 0,
+            stdout:
+            '<Configuration>\r\n<project test>\r\n</project>\r\n</Configuration>\r\nEffective POMs, after inheritance, interpolation, and profiles are applied:\r\n\r\n<!-- ====================================================================== -->\r\n<!--                                                                        -->\r\n<!-- Generated by Maven Help Plugin on 2017-06-29T09:43:41                  -->\r\n<!-- See: http://maven.apache.org/plugins/maven-help-plugin/                -->\r\n<!--                                                                        -->\r\n<!-- ====================================================================== -->\r\n\r\n<!-- ====================================================================== -->\r\n<!--                                                                        -->\r\n<!-- Effective POM for project                                              -->\r\n<!-- \'com.microsoft.xplatalm:xplatalmApp:jar:1.0-SNAPSHOT\'                  -->\r\n<!--                                                                        -->\r\n<!-- ====================================================================== -->\r\n\r\n<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">\r\n</project>\r\n\r\n[INFO] ------------------------------------------------------------------------\r\n[INFO] BUILD SUCCESS\r\n[INFO] ------------------------------------------------------------------------\r\n[INFO] Total time: 0.927 s\r\n[INFO] Finished at: 2017-06-29T09:43:41-04:00\r\n[INFO] Final Memory: 7M/18M\r\n[INFO] ------------------------------------------------------------------------'
+        },
+        '/home/bin/maven/bin/mvn -f pom.xml clean package': {
+            code: 0,
+            stdout: 'Maven package done'
+        },
+        '/home/bin/maven/bin/mvn -f CCReportPomA4D283EG.xml verify -Dmaven.test.skip=true': {
+            code: 0,
+            stdout: 'something'
+        }
+    },
+    findMatch: {
+        '**/TEST-*.xml': [
+            '/user/build/fun/test-123.xml'
+        ]
+    },
+    exist: {
+        [path.join(getTempDir(), '.mavenInfo')] : true,
+        [path.join('CCReport43F6D5EF', 'jacoco.xml')]: true,
+        'CCReportPomA4D283EG.xml': true
+    },
+    rmRF: {
+        target: { success: true },
+        CCReport43F6D5EF: { success: true },
+        'CCReportPomA4D283EG.xml': { success: true }
+    }
+};
+taskRunner.setAnswers(answers);
+
+const mockTaskLibClone = Object.assign({}, require('azure-pipelines-task-lib/mock-task'));
```

Comment:
Why do we need to mock it?

Reply:
I thought we needed this to mock TestPublisher and CodeCoveragePublisher classes, but it looks like they are already mocked in mock-task. I'll remove this